guild_scheduled_event_integration_test.go

```diff
@@ -0,0 +1,396 @@
+package disgord
+
+import (
+	"os"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var token string
+var guildID Snowflake
+
+var maxName = `Light it creeping a herb evening seas you're stars. 
+	Lights let dry land let female is blessed also blessed
+	they're life were rule that subdue, third may. Greater without, 
+	given can't And bring she'd created fruitful third sea. 
+	Good, dominion whose i blessed and second a appear replenish shall great, 
+	void two sea which god. A place female abundantly, 
+	seas fruitful moveth us heaven. Forth beginning to and image own seasons land had dry. 
+	Given that they're the face without. Wherein he first also. 
+	Fill hath. Sea. Have waters. Deep over earth grass fill had was it the a of.`
+
+func init() {
+	token = os.Getenv("DISCORD_BOT_TOKEN")
+	gID, _ := strconv.Atoi(os.Getenv("DISCORD_GUILD_ID"))
```

Comment:
use `token` and `guildAdmin.ID` from the client_integration_test.go instead. Then we have it all in one place, and we don't accidentally break integration tests by overwriting variables.

Reply:
this value doesn't share, I've to change it into a token from client_integration_test.go but still got an error.

Reply:
what was the error?

Reply:
`undeclared name: token`

Reply:
These values are in the same package but why it get `undeclared name:` 🤔